packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -105,137 +128,135 @@ export const App = ({
     }
   }, [metricsEnabled, config.sentry, config.version, loading, isSignedIn]);
 
-  useEffect(() => {
-    if (Storage.isLocalStorageEnabled(window)) {
-      let uniqueUserId = JSON.parse(
-        localStorage.getItem('__fxa_storage.currentAccountUid') || '{}'
-      );
+  return (
+    <Router basepath="/">
+      <AuthAndSetUpRoutes path="/*" />
+      <SettingsRoutes path="/settings/*" {...{ showRecoveryKeyV2 }} />
+    </Router>
+  );
+};
 
-      const accounts: Array<{ email: string; uid: string }> = JSON.parse(
-        localStorage.getItem('__fxa_storage.accounts') || '{}'
-      );
+const SettingsRoutes = ({
+  showRecoveryKeyV2,
+}: {
+  showRecoveryKeyV2?: boolean;
+} & RouteComponentProps) => {
```

Comment:
It is because we are passing `path` to it.
<img src="https://github.com/mozilla/fxa/assets/13018240/75bedb50-685c-4a40-97dc-7ea7afd1ca3d" width="500">